agent/lib/kontena/rpc/agent_api.rb

```diff
@@ -1,3 +1,5 @@
+require_relative '../models/node'
```

Comment:
Should these `models/...` requires be in the `kontena-agent.rb`, where everything else is required?